### ISSUE TITLE
Accessibility - Drupal and Dataverse - Color schema reference 

### ISSUE THREAD

**Issue by adam3smith** [QualitativeDataRepository/dataverse#76]
This is for reference only, can be close once we're done with everything.
Overall goal is WCAG AA compliance, which requires 4.5 :1 contrast for all text.

Text and element colors:
QDR Blue: #0039F5 (was #4B74FF)
Off Black: #333333 (unchanged)
Dark Grey: #424242 (was #767676; can be used on all backgrounds)
Medium Grey: #666666 (was #969696; can be used on white and backgrounds A & B)
Light Grey: #D7D7D7 (unchanged; can only be used inside elements in QDR Blue, Off-Black, or Dark Grey)

Background Colors (all unchanged)
Pure White: #FFFFFF
Section A: #F6F6F8
Section B: #E8E8ED
Section C: #D5D5DA
